packages/react/src/components/DataGrid/DataList.spec.tsx

```diff
@@ -0,0 +1,138 @@
+    }));
+    render(
+      <CunninghamProvider>
+        <DataList
+          columns={[{ field: "firstName" }, { field: "lastName" }]}
+          rows={rows}
+        />
+      </CunninghamProvider>
+    );
+
+    const table = screen.getByRole("table");
+    expect(queryAllByRole(table, "columnheader")).toEqual([]);
+
+    rows.forEach((row) => {
+      const element = screen.getByTestId(row.id);
+      const tds = getAllByRole(element, "cell");
+      expect(tds.length).toBe(2);
+      expect(tds[0].textContent).toEqual(row.firstName);
+      expect(tds[1].textContent).toEqual(row.lastName);
+    });
+  });
+  it("should render a DataList with working rows selection", async () => {
+    const rows = Array.from(Array(3)).map((_value, index) => ({
+      id: `list key for element ${index}`,
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+    }));
+
+    let lastRowSelection: Record<string, boolean> = {};
+
+    const Wrapper = () => {
+      const [rowSelection, setRowSelection] = useState({});
+      lastRowSelection = rowSelection;
+      return (
+        <CunninghamProvider>
+          <DataList
+            columns={[{ field: "firstName" }, { field: "lastName" }]}
+            rows={rows}
+            enableRowSelection={true}
+            rowSelection={rowSelection}
+            onRowSelectionChange={setRowSelection}
+          />
+        </CunninghamProvider>
+      );
+    };
+
+    render(<Wrapper />);
+
+    const user = userEvent.setup();
+    const rowsToSelect = [rows[0], rows[2]];
+
+    // Check first row.
+    let element = screen.getByTestId(rowsToSelect[0].id);
+    let checkbox: HTMLInputElement = getByRole(element, "checkbox");
+    await user.click(checkbox);
+    expect(lastRowSelection[rowsToSelect[0].id]).toBe(true);
+
+    // Check second row.
+    element = screen.getByTestId(rowsToSelect[1].id);
+    checkbox = getByRole(element, "checkbox");
+    await user.click(checkbox);
+    expect(lastRowSelection[rowsToSelect[0].id]).toBe(true);
+    expect(lastRowSelection[rowsToSelect[1].id]).toBe(true);
+
+    // Uncheck first row.
+    element = screen.getByTestId(rowsToSelect[0].id);
+    checkbox = getByRole(element, "checkbox");
+    await user.click(checkbox);
+    expect(lastRowSelection[rowsToSelect[0].id]).toBe(undefined);
+    expect(lastRowSelection[rowsToSelect[1].id]).toBe(true);
+  });
+
+  it("should render an empty grid", async () => {
+    const rows: Row[] = [];
+    render(
+      <CunninghamProvider>
+        <DataList
+          columns={[
+            {
+              field: "firstName",
+              headerName: "First name",
+            },
+          ]}
+          rows={rows}
+        />
+      </CunninghamProvider>
+    );
+
+    screen.getByRole("img", { name: /illustration of an empty table/i });
+    screen.getByText(/this table is empty/i);
+  });
+  it("should render a loading grid even if rows are empty", async () => {
```

Comment:
IMO this test is redundant as it is already test in `SimpleDataGrid.spec.tsx`, WDYT?